### ISSUE TITLE
sometime got high resistance

### ISSUE THREAD

**Issue by FestumZ** [adafruit/Adafruit_MAX31856#18]
I using mega2560 and connect six max31865 modules to detect PT100 temperature.
Sometimes read high resistance from different module....
How to fix this issues? 

thanks

**Comment by FestumZ** [comment]
High resistance means 998 ° C after conversion temperature reading...

**Comment by ladyada** [comment]
did you purchase the max31865 module from adafruit?

**Comment by FestumZ** [comment]
Yes, I bought the max31865 module from the distributor and the product has the same logo as your avatar on it.

**Comment by siddacious** [comment]
@FestumZ this is the MAX31856 library repo. Can you please confirm which sensor and which library you are using?

**Comment by FestumZ** [comment]
@siddacious I use the MAX31865 module and the Adafruit_MAX31865.h library.
